plugins/modules/ovirt_disk.py

```diff
@@ -579,8 +578,8 @@ def download_disk_image(connection, module):
 
 
 def upload_disk_image(connection, module):
-    transfers_service = connection.system_service().image_transfers_service()
-    hosts_service = connection.system_service().hosts_service()
+    connection.system_service().image_transfers_service()
+    connection.system_service().hosts_service()
```

Comment:
Why do we need to fetch those services when they are not used anywhere?

Reply:
Same answer as above.
Currently I have a limited availability to start testing this, but if you / @mnecas are sure it's safe, I can remove those calls at all :)

Reply:
If it is not used anywhere than it can be removed. Looks like it is some leftover code from the manual upload/download of disks.

Reply:
Done :)